docs/api-reference/extensions/v1beta1/definitions.html

```diff
@@ -3057,7 +3057,7 @@ <h3 id="_v1beta1_jobcondition">v1beta1.JobCondition</h3>
 <tbody>
 <tr>
 <td class="tableblock halign-left valign-top"><p class="tableblock">type</p></td>
-<td class="tableblock halign-left valign-top"><p class="tableblock">Type of job condition, currently only Complete.</p></td>
+<td class="tableblock halign-left valign-top"><p class="tableblock">Type of job condition.</p></td>
```

Comment:
No, I just had an eye in jobConditions in order to implement deploymentConditions, realized that that comment was wrong and fixed it. I can submit a separate PR for it.